Dockerfile

```diff
@@ -26,8 +26,6 @@ RUN if [ ! -f $CACHITO_ENV_FILE ]; then go mod download ; fi
 # Build manager
 RUN if [ -f $CACHITO_ENV_FILE ] ; then source $CACHITO_ENV_FILE ; fi ; CGO_ENABLED=0  GO111MODULE=on go build ${GO_BUILD_EXTRA_ARGS} -a -o ${DEST_ROOT}/manager main.go
 
-RUN cp -r templates ${DEST_ROOT}/templates
-
```

Comment:
Yes, thanks for removing this since it's not used and we'd have to `mkdir templates` just to make it happy. Better to not have it in the make file since i'ts not needed.